RctGcs/ui/display.py

```diff
@@ -3,20 +3,17 @@
 import logging
 import queue as q
 from functools import partial
-from pathlib import Path
-import RctGcs.config as config
-
-import RctGcs.rctCore
 
 import utm
```

Comment:
Imports not sorted

Reply:
These should be here per https://peps.python.org/pep-0008/#imports